qa/L1_jupyter_plugins/test.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash -e
+
+source ../setup_test.sh
```

Comment:
Long run I would make this sourced inside `test_template.sh`.

Reply:
Unfortunately, not all `test.sh` scripts use `test_template.sh` (i.e. defining test in `test_body` function). It could be unified, but is out of scope of this PR